packages/fluentui/docs/src/components/ComponentDoc/PerfChart/useBundleSizeData.ts

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import PerfDataContext from './PerfDataContext';
+
+const useBundleSizeData = (filter?: string) => {
```

Comment:
I suggest to switch later to [`react-query`](https://github.com/tannerlinsley/react-query) and slice data

![image](https://user-images.githubusercontent.com/14183168/77338497-bcb11f80-6d2a-11ea-9f89-4569c2614f66.png)
![image](https://user-images.githubusercontent.com/14183168/77338521-c6d31e00-6d2a-11ea-95fc-3b17e2fbe549.png)

Currently it takes 1 min to get an answer from backend and around 1 min to fetch data 💣